src/lodf_calculations.jl

```diff
@@ -109,7 +109,7 @@ function _calculate_LODF_matrix_KLU(
     return lodf_t
 end
 
-# ! temp for evaluation
+# ! used only in testing, can be removed in the future
```

Comment:
What's the point of leaving this method here? If it isn't going to be part of the standard release is better to remove it at once.